app/styles/components/_global.scss

```diff
@@ -55,6 +55,7 @@ button {
   }
   &.publish, &.unpublish {
     background-color: $ilios-green;
+    font-size: .80em;
```

Comment:
All other font sizes are specified as $base-font-size * .8 so if we ever change the base they will stay relatively the same.